README.md

```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images [hosted on Docker Hub](https://hub.docker.com/u/library). The main tenets are:
+
+- Support [multiple architectures](#architectures-other-than-amd64)
 
-## What do you mean by "Official"?
+- Exemplify [`Dockerfile` best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
 
-In many cases, the images in this program are not only supported but also *maintained* directly by the relevant upstream projects.
+- [Actively rebuilt](#library-definition-files) for updates and security fixes
```

Comment:
```suggestion
- [Actively rebuild](#library-definition-files) for updates and security fixes
```

Something felt off on this point; it turns out that the tense was different from the rest of the list.